packages/notifier/src/storesub.js

```diff
@@ -79,3 +80,41 @@ export const makeStoredSubscription = (
   return storesub;
 };
 harden(makeStoredSubscription);
+
+/**
+ * @template T
+ * @typedef {object} StoredPublisherKit
+ * @property {StoredSubscription<T>} subscriber TODO: change to StoredSubscriber<T> when available
+ * @property {IterationObserver<T>} publisher
+ */
+
+/**
+ * @template [T=unknown]
+ * @param {ERef<StorageNode>} [storageNode]
+ * @param {ERef<Marshaller>} [marshaller]
+ * @param {string} [childPath]
+ * @returns {StoredPublisherKit<T>}
+ */
+export const makeStoredSubscriptionKit = (
```

Comment:
Please change the name.

```suggestion
export const makeStoredPublisherKit = (
```